seqr/views/utils/orm_to_json_utils.py

```diff
@@ -205,7 +204,7 @@ def _get_pedigree_image_url(pedigree_image):
                 pedigree_image = pedigree_image.url
             except Exception:
                 pedigree_image = None
-        return os.path.join("/media/", pedigree_image) if pedigree_image else None
+        return pedigree_image
```

Comment:
@hanars I confirmed that this works in dev -- 

When the GCS bucket is enabled, the URL returned is a google object URL to the image in the bucket, and when it's disabled we're getting a `/media/foobar/baz.jpg` image as expected. Prepending the `/media` was redundant.